core/object.cpp

```diff
@@ -1315,9 +1315,10 @@ Error Object::connect(const StringName &p_signal, const Callable &p_callable, co
 
 	Callable target = p_callable;
 
-	if (s->slot_map.has(target)) {
+	//compare with the base callable, so binds can be ignored
```

Comment:
Please try to get used to this comment style that the team agreed upon:
```suggestion
	// Compare with the base callable, so binds can be ignored.
```

We can't rely on formatting tools for that, we have to be consistent in the code we write.

Reply:
The problem is that this comment style is everywhere in the codebase, even in that file. I honestly just forget. We should try to ask the clang format guys to add support for this and pester them about it.